src/HasStateTransitions.php

```diff
@@ -47,7 +47,7 @@ public function transitionStateTo(TransitionState $to, string $field = null): se
     /**
      * @return array<\LenderSpender\StateTransitionWorkflow\TransitionState>
```

Comment:
Fixing this generic by specifying the key as a first argument will possibly resolve some ignored PHPStan errors.